sdk/storage/azure-storage-blobs/inc/azure/storage/blobs.hpp

```diff
@@ -1,12 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+/**
+ * @brief Includes all public headers from Azure Storage Blobs SDK library.
+ *
+ */
+
 #pragma once
 
+#include "azure/storage/blobs/protocol/blob_rest_client.hpp"
+
 #include "azure/storage/blobs/append_blob_client.hpp"
 #include "azure/storage/blobs/blob_client.hpp"
 #include "azure/storage/blobs/blob_container_client.hpp"
 #include "azure/storage/blobs/blob_lease_client.hpp"
+#include "azure/storage/blobs/blob_options.hpp"
+#include "azure/storage/blobs/blob_responses.hpp"
```

Comment:
```suggestion
```

Reply:
This is internal implementation and should not be exposed directly to customers.